### PR TITLE
chore: Add "Two Minute Papers" in ML in YouTube

### LINKED ISSUE #762: [Add] A YouTube channel called "Two Minute Papers"
### Site URL

https://youtube.com/@TwoMinutePapers

### Description of the above provided site

This is an interesting channel explaining a lot of Research Papers relating to ML, Data Science, and AI. All the research papers are explained in english. They are presented in an interesting way.

### Category

Youtube

### Sub Category

ML, AI, Data Science

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by AkhilProto**
Closes #762  

## Changes proposed

Added a YouTube channel named "Two Minute Papers". This channel covers a detailed explanation of many Artificial Intelligence and Machine Learning Research Papers. The explanation is neat, well-detailed, interesting and you would be watching the whole video without getting bored! The channel provides the visuals of the Model/Application in work while explanation. At last, I would always recommend this channel to anyone interested in AI and ML.

## Screenshots
Screenshot_2023-05-27_22_46_49

### CODE REVIEW

File: database/youtube/machine-learning.json [hunk 1]
```diff
@@ -1,12 +1,19 @@
...
-    {
-        "name":"Udacity",
-        "description":"Free online course for learning tensorflow for Deep learning",
-        "url":"https://learn.udacity.com/courses/ud187",
-        "category":"online platform",
-        "subcategory":"tensorflow",
-        "language":"english"
+    {
+        "name": "Two Minute Papers",
+        "description": "Artificial Intelligence and Machine Learning Research Papers explained Interestingly and in a Well-Detailed Manner. What a Time to be Alive!",
```
Comment (CBID2) [inline database/youtube/machine-learning.json:4]: [Proofreading] Just some typos and unclear parts 
```suggestion
        "description": "These research papers discuss Artificial Intelligence and Machine Learning in an engaging and detailed manner. What a Time to be Alive!",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @AkhilProto! I just pointed out some proofreading errors in my inline comments. In the future, consider running your descriptions through spellcheck or Grammarly before submitting them to your PR.

File: database/youtube/machine-learning.json [hunk 2]
```diff
@@ -25,36 +32,35 @@
...
-     },
-    {
-        "name":"Google",
-        "description":"Free online course provided by google to learn machine learning and it's implementations",
-        "url":"https://developers.google.com/machine-learning",
-        "category":"online platform",
-        "subcategory":"machine-learning",
-        "language":"english"
+        "name": "Krish Naik",
+        "description": "This channel provides videos on Machine Learning, Deep Learning and Artificial Intelligence with real-world problem scenarios.",
```
Comment (CBID2) [inline database/youtube/machine-learning.json:36]: [Proofreading] 
```suggestion
        "description": "This channel provides videos on Machine Learning, Deep Learning, and Artificial Intelligence with real-world problems.",
```

File: database/youtube/machine-learning.json [hunk 3]
```diff
@@ -25,36 +32,35 @@
...
+        "url": "https://www.youtube.com/user/krishnaik06",
+        "category": "youtube",
+        "subcategory": "machine-learning",
+        "language": "english"
+    },
+    {
+        "name": "Google",
+        "description": "Free online course provided by google to learn machine learning and it's implementations",
```
Comment (CBID2) [inline database/youtube/machine-learning.json:44]: [Proofreading]
```suggestion
        "description": "Free online course provided by Google to learn Machine Learning and it's implementations",
```

File: database/youtube/machine-learning.json [hunk 4]
```diff
@@ -25,36 +32,35 @@
...
     },
     {
-        "name":"FreeCodeCamp",
-        "description":"Learn Machine Learning in a way that is accessible to absolute beginners. You will learn the basics of Machine Learning and how to use TensorFlow to implement many different concepts.",
-        "url":"https://www.youtube.com/watch?v=i_LwzRVP7bg&t=769s&ab_channel=freeCodeCamp.org",
-        "category":"youtube",
-        "subcategory":"machine-learning",
-        "language":"english"
+        "name": "FreeCodeCamp",
+        "description": "Learn Machine Learning in a way that is accessible to absolute beginners. You will learn the basics of Machine Learning and how to use TensorFlow to implement many different concepts.",
```
Comment (CBID2) [inline database/youtube/machine-learning.json:52]:
```suggestion
        "description": "Here, you will learn the basics of Machine Learning and how to use TensorFlow in different scenarios.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @AkhilProto! :) I pointed out more proofreading errors in my inline comments. Like I mentioned in my previous review, consider running your descriptions through Grammarly or Spellcheck before making a pull request. Keep going! :)

**Review by CBID2 — APPROVED** [review]
Great job @AkhilProto! 😄 LGTM! 😄

File: database/youtube/machine-learning.json (hunk 1 above)
Comment (AkhilProto) [reply database/youtube/machine-learning.json]: Just wanna suggest a small change here. I think it should be **This channel discusses Artificial Intelligence and Machine Learning Research Papers in an interesting and well-detailed manner**

**Review by AkhilProto — COMMENTED** [review]
I have commited all of the changes you requested except the description of "Two Minute Papers". I think the Research Papers don't discuss, the channel does.

**Comment by CBID2** [comment]
> I have commited all of the changes you requested except the description of "Two Minute Papers". I think the Research Papers don't discuss, the channel does.

Ok

**Review by CBID2 — APPROVED** [review]
Great job @AkhilProto! 😄 LGTM! 😄

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**